tests/support/controller.rs

```diff
@@ -15,6 +15,10 @@ pub fn new() -> Controller {
     Controller::new()
 }
 
+pub fn new_unordered() -> Controller {
```

Comment:
Was this added for the case of an applied traffic split when we don't know which leaf service the request is routed to, but do know that which ever one was chosen there should be a destination lookup for it?